arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/signals/Signals.java

```diff
@@ -142,20 +142,23 @@ public void prepare(MessageFrame frame, PlatformController platformController, H
 
     switch (opCode) {
       case CALLDATACOPY, CODECOPY -> {
-        this.mxp = ex.outOfMemoryExpansion() || ex.outOfGas() || ex.none();
+        this.mxp =
+            ex == Exceptions.OUT_OF_MEMORY_EXPANSION || ex == Exceptions.OUT_OF_GAS || ex.none();
```

Comment:
Maybe add parenthesis to make it more readable. 
Also, why not use Exceptions.NONE instead of ex.none() for more readability ?